src/raft.c

```diff
@@ -343,21 +343,41 @@ RedisModuleUser *RaftGetACLUser(RedisModuleCtx *ctx, RedisRaftCtx *rr, RaftRedis
     return user;
 }
 
+void handleUnblock(RedisModuleCtx *ctx, RedisModuleCallReply *reply, void *private_data)
+{
+    UNUSED(ctx);
+
+    BlockedCommand *bc = private_data;
+    if (bc->req) {
+        RedisModule_ReplyWithCallReply(bc->req->ctx, reply);
+        RaftReqFree(bc->req);
+    }
+
+    RedisModule_FreeCallReply(reply);
+    deleteBlockedCommand(bc->idx);
+    freeBlockedCommand(bc);
```

Comment:
seems like you can combine these two

Reply:
I wanted to make them parallel alloc/add